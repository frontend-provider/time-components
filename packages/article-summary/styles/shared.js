const sharedStyles = {
  text: {
    color: "#696969",
    fontSize: 14,
    fontFamily: "TimesDigitalW04",
    lineHeight: 20,
    marginBottom: 10,
    flexWrap: "wrap"
  },
  metaText: {
    color: "#696969",
    fontSize: 13,
    lineHeight: 15,
    fontFamily: "GillSansMTStd-Medium",
    marginBottom: 5
  },
  labelWrapper: {
    marginBottom: 0
  },
  headline: {
    color: "#333333",
    marginBottom: 5,
    fontFamily: "TimesModern-Bold",
    fontWeight: "400"
  },
  headlineWrapper: {
    fontSize: 22,
    lineHeight: 22
  }
};

export default sharedStyles;
