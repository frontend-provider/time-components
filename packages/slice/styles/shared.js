import { StyleSheet } from "react-native";

const sharedStyle = {
  container: {
    marginTop: 10,
    paddingLeft: 10,
    paddingRight: 10
  },
  titleContainer: {
    borderStyle: "solid",
    borderBottomColor: "#dbdbdb",
    borderBottomWidth: StyleSheet.hairlineWidth,
    borderTopColor: "#dbdbdb",
    borderTopWidth: StyleSheet.hairlineWidth,
    height: 57,
    display: "flex",
    alignItems: "center",
    justifyContent: "center"
  },
  title: {
    fontFamily: "TimesModern-Bold",
    fontSize: 26,
    color: "#333"
  },
  cardContainer: {
    paddingBottom: 10,
    paddingTop: 10
  },
  headline: {
    color: "#333333",
    marginBottom: 5,
    fontFamily: "TimesModern-Bold",
    fontWeight: "400"
  }
};

export default sharedStyle;
