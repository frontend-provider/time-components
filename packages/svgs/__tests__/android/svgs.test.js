import shared from "../shared";

describe("Svgs test on android: ", () => {
  shared();
});
