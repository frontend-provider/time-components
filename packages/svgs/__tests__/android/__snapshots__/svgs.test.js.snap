// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Svgs test on android:  renders correctly 1`] = `
<ARTSurfaceView
  style={
    Object {
      "height": 111.11111111111111,
      "width": 100,
    }
  }
>
  <ARTGroup
    opacity={1}
    transform={
      Array [
        0.9259259259259259,
        0,
        0,
        0.9259259259259259,
        -134.25925925925927,
        -46.2962962962963,
      ]
    }
  >
    <ARTGroup
      opacity={1}
      transform={
        Array [
          1,
          0,
          0,
          1,
          0,
          0,
        ]
      }
    >
      <ARTShape
        d={
          Array [
            0,
            211.26076,
            54,
            2,
            211.231367,
            54,
            2,
            147.67512,
            54,
            2,
            145,
            85.7081465,
            2,
            146.922096,
            86.4489102,
            3,
            146.922096,
            86.4489102,
            164.867589,
            68.1355181,
            168.301115,
            65.0001546,
            3,
            171.728017,
            61.8689133,
            174.237132,
            61.0885763,
            176.436179,
            60.3527593,
            3,
            180.998206,
            59.169681,
            185.977937,
            59.2150255,
            185.977937,
            59.2150255,
            2,
            186.109581,
            59.2150255,
            2,
            186.109581,
            156.560932,
            2,
            169.259886,
            164.473953,
            2,
            169.259886,
            166,
            2,
            228.735147,
            166,
            2,
            228.735147,
            164.473953,
            2,
            211.889177,
            156.560932,
            2,
            211.889177,
            59.2150255,
            2,
            212.01751,
            59.2150255,
            3,
            212.01751,
            59.2150255,
            216.992272,
            59.169681,
            221.558854,
            60.3527593,
            3,
            223.757072,
            61.0885763,
            226.266601,
            61.8689133,
            229.691848,
            65.0001546,
            3,
            233.130341,
            68.1355181,
            251.071695,
            86.4489102,
            251.071695,
            86.4489102,
            2,
            253,
            85.7081465,
            2,
            250.317842,
            54,
            2,
            211.270695,
            54,
            2,
            211.242545,
            54,
          ]
        }
        fill={
          Array [
            0,
            0,
            0,
            0,
            1,
          ]
        }
        opacity={1}
        stroke={null}
        strokeCap={1}
        strokeDash={null}
        strokeJoin={1}
        strokeWidth={1}
        transform={
          Array [
            1,
            0,
            0,
            1,
            0,
            0,
          ]
        }
      />
    </ARTGroup>
  </ARTGroup>
</ARTSurfaceView>
`;
